nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -86,6 +89,25 @@ def _sanitize_k8s_object(cls, data):
         else:
             return data
 
+    @staticmethod
+    def _list_helm_releases():
+        cmd = 'helm list --all-namespaces -o json'
+        try:
+            result = run(cmd, capture_output=True,
+                         timeout=10, encoding='UTF-8',
+                         shell=True, check=False)
+            if result.returncode != 0:
+                log.error('Failed to list Helm releases: %s', result.stderr)
+                return []
```

Comment:
That could be handled in an exception by setting `check=True`:
```python
except CalledProcessError as e:
    log.error('Failed to list Helm releases: %s', e.stderr)
```

Reply:
I prefer a simple and more efficient solution with just checking the return code. Exception is a costly thing, and here brings no additional advantage in clarity or anything else.